tools/iotools_test.go

```diff
@@ -0,0 +1,76 @@
+package tools_test
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/github/git-lfs/errors"
+	. "github.com/github/git-lfs/tools"
```

Comment:
As @rubyist points out, on second thought, this is not such a great idea. Having dot-imports makes it hard for humans and tools to figure out where identifiers are coming from. We already isolate ourselves from private members, which is all that matters 😄